library/Icingadb/Widget/Detail/MultiselectQuickActions.php

```diff
@@ -158,7 +159,7 @@ protected function assemble()
             $this->assembleAction(
                 'processCheckresult',
                 t('Process check result'),
-                'icon-edit',
+                'random',
```

Comment:
Isn't that the icon that's also being used for indicating a flapping host/service? (If not exactly this icon, it's at least a very similar one)

Personally, I'd leave this the same as before, but change the icon for the acknowledge action. (Maybe the check icon or so)

Reply:
Sounds good. I'll change it.

Reply:
✅